### PR TITLE
Enable CoffeeScript

### PR DESCRIPTION
**Pull request by salbertson**

### CODE REVIEW

File: .hound.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+ruby:
+  enabled: true
+
+coffee_script:
+  enabled: true
```
Comment (croaky) [inline .hound.yml]: Also SCSS?

**Comment by croaky** [comment]
Good to merge.

**Comment by gylaz** [comment]
Why CoffeeScript? Why not all the languages?

**Comment by salbertson** [comment]
@gylaz, because it came up [here](https://github.com/thoughtbot/hound/pull/575#diff-for-comment-23513806). We can enable other stuff in a different PR.

**Pull request merged**